source/conf.py

```diff
@@ -115,6 +115,7 @@
 # a list of builtin themes.
 #
 html_theme = 'bootstrap'
+html_permalinks_icon = ''
```

Comment:
This is NOT related to the rest of the patch.

Reply:
Hi @segfaultxavi. 
It is related to the rest of the patch. The new Algolia crawler is indexing ¶ which is added by Sphinx by default as anchor text to an inside h1 (html_permalinks_icon by default is ¶). For example:
```
<h1>Setting up your workstation<a class="headerlink" href="#setting-up-your-workstation" title="Permalink to this headline">¶</a></h1>
```
You can check in the preview search https://deploy-preview-779--nemtech.netlify.app/:

![image](https://user-images.githubusercontent.com/17684281/170538126-9ed21e4a-618d-44a0-a658-03456eb4eb93.png)

The old Crawler configuration (or default behavior) indexed it as whitespace (you can still check in docs.symbol.dev Algolia API results when you type something in the search bar).

So instead of changing Crawler configuration (to which at that point I had no access) I removed this ¶ since it's redundant anyway (correct icon is added using CSS):

https://github.com/symbol/symbol-docs/blob/main/source/_static/css/custom.css#L298

I hope it explains this change.

Reply:
Ooooohhhh... silly crawler :)
I thought you had removed the permalink completely. Thanks for the detailed explanation!